keyboards/ut472/keymaps/via/config.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2018 Carlos Filoteo
```

Comment:
This file should be removed

Reply:
thank you. these two issues should now be resolved.